src/libcore/macros.rs

```diff
@@ -550,15 +550,15 @@ macro_rules! unimplemented {
 /// into libsyntax itself.
 ///
 /// For more information, see documentation for `std`'s macros.
-#[cfg(dox)]
-pub mod builtin {
+mod builtin {
```

Comment:
This looks wrong, doesn't removing `pub` affect the docs? I could be wrong though.